### PR TITLE
[AMORO-3192] Add the "clean-orphan-file.interval-time-minutes" parameter

### LINKED ISSUE #3192: [Improvement]: Add the "clean-orphan-file.interval-time-minutes" parameter.
### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

By adding the parameter "clean-orphan-file.interval-time-minutes" to control the frequency of cleaning orphan files, users can adjust the cleaning frequency as needed instead of a fixed 24-hour interval.

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lintingbin**
## Why are the changes needed?

Close #3192.

## Brief change log

- Add a parameter to replace the INTERVAL constant, which was previously fixed at 24 hours in the OrphanFilesCleaningExecutor class.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? yes
- If yes, how is the feature documented? docs

### CODE REVIEW

**Review by tcodehuber — COMMENTED** [review]
Thanks for your work, but I think the default check interval should be less than the default min existing time.

**Comment by lintingbin** [comment]
> Thanks for your work, but I think the default check interval should be less than the default min existing time.

Why is this necessary? The "min existing time" ensures that useful files are not mistakenly deleted, so setting it to two to three days is usually sufficient. However, cleaning orphan files is a performance-intensive task, so it's generally appropriate to configure the cleanup to occur once a week or once a month.

**Comment by zhoujinsong** [comment]
@lintingbin Thanks for the contribution.

The scheduling interval for the current expiration task is usually set as an AMS configuration like `data-expiration.interval`, which applies to all tables. I think this meets the requirements, as we generally do not adjust different execution interval for different tables. What do you think?

BTW, we are improving the configuration value for time duration with unit support like 3d, we can do it for the new configuration.

**Comment by zhoujinsong** [comment]
> Why is this necessary? The "min existing time" ensures that useful files are not mistakenly deleted, so setting it to two to three days is usually sufficient. However, cleaning orphan files is a performance-intensive task, so it's generally appropriate to configure the cleanup to occur once a week or once a month.

I agree with @lintingbin, there is no connection between the values of these two configurations.

**Comment by lintingbin** [comment]
> @lintingbin Thanks for the contribution.
> 
> The scheduling interval for the current expiration task is usually set as an AMS configuration like `data-expiration.interval`, which applies to all tables. I think this meets the requirements, as we generally do not adjust different execution interval for different tables. What do you think?
> 
> BTW, we are improving the configuration value for time duration with unit support like 3d, we can do it for the new configuration.

The modifications have been completed according to the suggestions.

File: docs/user-guides/configurations.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Data-cleaning configurations are applicable to both Iceberg Format and Mixed str
...
 | snapshot.keep.min-count                     | 1                | Minimum number of snapshots retained for table expiration                                                                                                                                                                                                             |
 | clean-orphan-file.enabled                   | false            | Enables periodically clean orphan files                                                                                                                                                                                                                               |
+| clean-orphan-file.interval                  | 1d               | The interval for Cleaning orphan files within a specified time                                                                                                                                                                                                        |
```
Comment (zhoujinsong) [inline docs/user-guides/configurations.md:75]: This change may need to be rollback

And you may want to add this new configuration example into `dist/src/main/amoro-bin/conf/config.yaml`

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**